doc/X-OR_tutorial/X-OR_tutorial.rst

```diff
@@ -712,9 +712,9 @@ And this maximum is:
 
 If this is not clear enough you can take a look at the edge labels in the above graph.
 
-To choose a base for a tree we need to apply the previous algorithm to all the roots of tree.
+To choose a base for a tree we, need to apply the previous algorithm to all the roots of tree.
```

Comment:
I don't see how this configuration could happen... You're doing the obfuscation of one XOR operation (chained or not), if there are other XOR using operands previously encountered, you will just consider it as another tree, no ?